contrib/largeNbDicts/largeNbDicts.c

```diff
@@ -485,6 +535,29 @@ static size_t compressBlocks(size_t* cSizes,   /* optional (can be NULL). If pre
 
 /* ---  Benchmark  --- */
 
+typedef struct {
+    ZSTD_CCtx* cctx;
+    size_t nbDicts;
+    size_t dictNb;
+    cdict_collection_t dictionaries;
+} compressInstructions;
+
+compressInstructions createCompressInstructions(cdict_collection_t dictionaries)
+{
+    compressInstructions ci;
+    ci.cctx = ZSTD_createCCtx();
+    assert(ci.cctx != NULL);
```

Comment:
`CONTROL()`

Reply:
And yes, here also, `CONTROL()` is preferable, since nothing handles the case `ci.cctx == NULL` afterwards.